scripts/license-check.sh

```diff
@@ -102,6 +102,10 @@ check_copyright_headers() {
     ! -name '*.grpc.swift' \
     ! -name 'LinuxMain.swift' \
     ! -name 'XCTestManifests.swift' \
+    ! -path './FuzzTesting/.build/*' \
+    ! -path './Performance/QPSBenchmark/.build/*' \
+    ! -path './Performance/Benchmarks/.build/*' \
+    ! -path './scripts/.swift-format-source/*' \
```

Comment:
This is now necessary because the soundness check first runs the license check and _then_ would check out and build swift-format, meaning that it'd check the headers of swift-format. This wasn't an issue because of the ordering.